app/src/main/java/com/kickstarter/viewmodels/CreatorDashboardHeaderHolderViewModel.java

```diff
@@ -0,0 +1,112 @@
+package com.kickstarter.viewmodels;
+
+
+import android.support.annotation.NonNull;
+import android.util.Pair;
+
+import com.kickstarter.libs.ActivityViewModel;
+import com.kickstarter.libs.Environment;
+import com.kickstarter.libs.RefTag;
+import com.kickstarter.libs.rx.transformers.Transformers;
+import com.kickstarter.libs.utils.NumberUtils;
+import com.kickstarter.libs.utils.ProjectUtils;
+import com.kickstarter.models.Project;
+import com.kickstarter.models.ProjectStats;
+import com.kickstarter.ui.viewholders.CreatorDashboardHeaderViewHolder;
+import com.kickstarter.viewmodels.inputs.CreatorDashboardHeaderHolderViewModelInputs;
+import com.kickstarter.viewmodels.outputs.CreatorDashboardHeaderHolderViewModelOutputs;
+
+import rx.Observable;
+import rx.subjects.BehaviorSubject;
+import rx.subjects.PublishSubject;
+
+public interface CreatorDashboardHeaderHolderViewModel {
+
+  final class ViewModel extends ActivityViewModel<CreatorDashboardHeaderViewHolder> implements
+    CreatorDashboardHeaderHolderViewModelInputs, CreatorDashboardHeaderHolderViewModelOutputs {
+
+
+    public ViewModel(final @NonNull Environment environment) {
+      super(environment);
+
+      projectAndStats
+        .map(ps -> ps.first)
```

Comment:
we have a PairUtils class so we can go point free with this, e.g. `.map(PairUtils::first)`